inbox/api/filtering.py

```diff
@@ -144,11 +147,6 @@ def threads(namespace_id, subject, from_addr, to_addr, cc_addr, bcc_addr,
         expand = (view == 'expanded')
         query = query.options(*Thread.api_loading_options(expand))
 
-    query = query.order_by(desc(Thread.recentdate)).limit(limit)
-
-    if offset:
-        query = query.offset(offset)
-
     if view == 'ids':
         return [x[0] for x in query.all()]
 
```

Comment:
What happened to the second join to get the message_ids and categories to show up? I remember this not working properly without that there.